qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/helper/TestData.java

```diff
@@ -684,4 +685,14 @@ public String toString() {
 		public String datetime;
 		public String uuid;
 	}
+	
+	public static String createRandomName(){
+		int lengthOfRandomString = 5;
+		String characters = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz!@#$%&";
+        Random random = new Random();
+        StringBuffer stringBuffer = new StringBuffer(lengthOfRandomString);
+		for (int i = 0; i < lengthOfRandomString; i++)
+			stringBuffer.append(characters.charAt(random.nextInt(characters.length())));
+		return stringBuffer.toString() + "test";
+	}
```

Comment:
why not take advantage of [RandomStringUtil.randomAlphabetic()](https://www.tabnine.com/code/java/methods/org.apache.commons.lang3.RandomStringUtils/randomAlphabetic) method to create a random string directly from the [test implementation](https://github.com/openmrs/openmrs-contrib-qaframework/blob/5074f9b3d09a3d69c1db89db08aa32312f6715d8/qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/RolesAndPrivilegesSteps.java)?